package.json

```diff
@@ -6,12 +6,18 @@
     "node": ">= 6.0.0",
     "npm": ">= 3.0.0"
   },
+  "scripts": {
+    "potools": "potools"
```

Comment:
This is a smart solution to getting to the bin files without needing to point to node_modules 👍